src/zed_host/zed_fake.bag

```diff
@@ -0,0 +1 @@
+https://purdue0-my.sharepoint.com/:u:/g/personal/wdobert_purdue_edu/ERy1XQnN5PlBlYDR4rpEfQgBGxSU9Xul077LRsxEBDT_5g?e=sfXLWy
```

Comment:
How big is it compared to git's file size limit? I think actually having the bag file in the repo is much better than having to download from OneDrive separately. IMO, making it less than a second would be better than having a longer one and downloading separately. @alanssitis and @Reschivon, what do you think?

Reply:
It's 304 MB, whereas GitHub's filesize limit is 100 MB.

Reply:
Hmm, now that I think about it, having a 100MB file would be pretty bad. Here are a few options I can think of that may not be a little bit more scalable than OneDrive: 
* Make another repo in AMP called 'bagfiles' and use [GitHub LFS](https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-git-large-file-storage).
* Put the file in either our Google Drive or on the webserver that our website runs on.

Thoughts?

Reply:
Not sure what pros/cons GitHub LFS has in regards to ease of use, but the AMP Google Drive would definitely be a better place to host it if we don't end up using LFS.

Reply:
@ihagedo knows a little about LFS so maybe he can chime in here.

If not LFS, then I think we need to put the file somewhere (ideally Drive) such that we could do something like `wget $(cat zed_and_rtab_map_bag_link)` to easily download the bagfile to avoid having to open a web browser.

Reply:
Also, could you please rename this to something like `zed_fake_bag_link` that way there is no confusion as to whether or not this is an actual bagfile?

Reply:
Thanks!